third_party/move/move-vm/types/src/values/values_impl.rs

```diff
@@ -488,7 +529,8 @@ impl ValueImpl {
             | (Address(_), _)
             | (Container(_), _)
             | (ContainerRef(_), _)
-            | (IndexedRef(_), _) => {
+            | (IndexedRef(_), _)
+            | (Delayed { .. }, _) => {
```

Comment:
Just curious. Why not use `(_, _)` here instead of the long condition?

Reply:
Using wildcard matching can be error prone, if you add a new enum variant, your match statements would fall into default behaviour, which might not be what you want.